scripts/assets/file_content_validation_checks.textproto

```diff
@@ -341,7 +341,7 @@ file_content_checks {
 }
 file_content_checks {
   file_path_regex: "app/src/main/res/layout.*?/.+?\\.xml"
-  prohibited_content_regex: "@color/(?!component_color_).+"
+  prohibited_content_regex: "@color/(?!component_color_).+|#\\p{XDigit}"
```

Comment:
@BenHenning I want to prohibit only Hex value of color, but `#\\p{XDigit}` consider TODO's also such as "TODO(#352):" 
So how we make this as prohibit in regex `android:background="#000000"`  ?
Even we not add extra `"` in regex?

Reply:
Thanks @BenHenning, I got the solution by `\"` we can use " in regex.